robots/flakefinder/main.go

```diff
@@ -86,23 +88,32 @@ func main() {
 		}
 	}
 
-	var c client = github.NewClient(secretAgent.GetTokenGenerator(o.token), o.graphqlEndpoint, o.endpoint.Strings()...)
-	query := MakeQuery("repo:kubevirt/kubevirt is:merged is:pr", o.merged, time.Now())
-	issues, err := c.FindIssues(query, "", false)
-	if err != nil {
-		log.Fatalf("Failed run: %v", err)
-	}
+	ctx := context.Background()
+	ts := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: string(secretAgent.GetSecret(o.token))},
+	)
+	tc := oauth2.NewClient(ctx, ts)
 
+	c := github.NewClient(tc)
+	query := MakeQuery("repo:kubevirt/kubevirt is:merged is:pr", o.merged, time.Now())
 	prs := []*github.PullRequest{}
-	for _, issue := range issues {
-		pr, err := c.GetPullRequest("kubevirt", "kubevirt", issue.Number)
+	for nextPage := 1; nextPage > 0; {
+		issues, response, err := c.Search.Issues(ctx, query, &github.SearchOptions{ListOptions: github.ListOptions{Page: nextPage}})
```

Comment:
the new client can do `c.PullRequests.List(...)`, any reason to stay with `c.Search.Issues(....)` and then get every single PR?

Reply:
AFAIU we need to search for *merged* PRs. I did not find out how to do this with `PullRequest.List`, or did I overlook something?

Reply:
OK, changed to use `PullRequests.List`.